code_samples/data_migration/src/Migrations/Step/ReplaceNameStepExecutor.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Migrations\Step;
+
+use eZ\Publish\Core\FieldType\TextLine\Value;
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Values\Filter\Filter;
+use Ibexa\Platform\Contracts\Migration\StepExecutor\AbstractStepExecutor;
+use Ibexa\Platform\Migration\ValueObject\Step\StepInterface;
+
+final class ReplaceNameStepExecutor extends AbstractStepExecutor
+{
+    private ContentService $contentService;
+
+    public function __construct(
+        ContentService $contentService
+    ) {
+        $this->contentService = $contentService;
+    }
+
+    protected function doHandle(StepInterface $step): void
+    {
+        assert($step instanceof ReplaceNameStep);
+
+        $contentItems = $this->contentService->find(new Filter());
+
+        foreach ($contentItems as $contentItem) {
+            $struct = $this->contentService->newContentUpdateStruct();
+
+            foreach ($contentItem->getFields() as $field) {
+                if ($field->fieldTypeIdentifier !== 'ezstring') {
+                    continue;
+                }
+
+                if ($field->fieldDefIdentifier === 'identifier') {
+                    continue;
+                }
+
+                if ($field->value == 'Company Name') {
+                    $struct->setField($field->fieldDefIdentifier, new Value($step->getReplacement()));
+                }
```

Comment:
```suggestion
                if ($field->value === 'Company Name') {
                    $struct->setField($field->fieldDefIdentifier, new Value($step->getReplacement()));
                }
```
We could also check & replace if value contains such a string, not only equals:
```suggestion
                if (str_contains($field->value, 'Company Name')) {
                    $newValue = str_replace('Company Name', $step->getReplacement(), $field->value);
                    $struct->setField($field->fieldDefIdentifier, new Value($newValue));
                }
```

(`str_contains` is a PHP8 only function, but it is contained in Symfony's PHP8 polyfill)